src/app/helpers/sideMenu.helper.ts

```diff
@@ -1,12 +1,30 @@
-export class SideMenuHelper{
-    constructor(private readonly menuPanel: HTMLElement){
-        this.menuPanel = menuPanel;
-    }
-    public openNav() : void{
-        this.menuPanel.style.width = "250px";
-      }
-      
-      public closeNav() : void {
-        this.menuPanel.style.width = "0";
-      }
-}
\ No newline at end of file
+// TODO: Refactor tp SideMenuComponent
+export class SideMenuHelper {
+private readonly menuPanel : HTMLElement;
```

Comment:
Please, check if we really need to store the reference to the element in the scope of helper?

Reply:
done